src/components/plans/ChargePercentage.tsx

```diff
@@ -192,25 +274,35 @@ const Container = styled.div`
   }
 `
 
+const Input = styled(TextInput)`
+  flex: 1;
+`
+
 const InputEnd = styled(Typography)`
   margin-right: ${theme.spacing(4)};
 `
 
 const LineAmount = styled.div`
   display: flex;
 
-  > *:first-child {
+  > *:not(:last-child) {
     margin-right: ${theme.spacing(3)};
-    flex: 1;
   }
 
   > *:last-child {
-    transform: translateY(50%);
+    margin-top: 34px;
```

Comment:
Hum... This is not from the spacing system 🤔

Reply:
Seen together,  the use case was a bit special and we decided to keep those values as they are